cap/modules/schemas/models.py

```diff
@@ -254,13 +269,102 @@ def add_read_access_for_all_users(self):
         )
         db.session.flush()
 
+    def modify_record_permissions(
+        self, permissions, record_type="deposit", schema_action="allow"
+    ):
+        allowed_actions = current_app.extensions['invenio-access'].actions
```

Comment:
We can load the allowed_actions in top of module or what's the reason we are loading in every method separately?